src/clj/game/cards/identities.clj

```diff
@@ -29,7 +29,49 @@
 
 ;;; Card definitions
 (def cards-identities
-  {"Adam: Compulsive Hacker"
+  {
+   "419: Amoral Scammer"
+   {:events {:corp-install
+             {:delayed-completion true
+              :req (req (first-event? state :corp :corp-install))
+              :effect
+              (req (show-wait-prompt state :corp "Runner to use 419: Amoral Scammer")
+                     (let [itarget target]
+                       (continue-ability
+                         state side
+                         {:optional
+                          {:prompt "Expose installed card unless Corp pays 1 [Credits]?"
+                           :player :runner
+                           :delayed-completion true
+                           :no-ability {:effect (req (clear-wait-prompt state :corp)
+                                                     (effect-completed state side eid))}
```

Comment:
This should be unneeded. The no-ability will assume the parent's eid, and will be automatically triggered as completed because the no-ability is not delayed completion.